docs/popup.js

```diff
@@ -168,6 +174,9 @@ function setUpPopup() {
       const closeButton = document.createElement('button');
       closeButton.className = 'popup-button';
       closeButton.innerHTML = 'close';
+      const closeIcon = document.createElement('i');
```

Comment:
Would you mind adding an icon for the save button, above?

Reply:
Ah missed that thanks